### PR TITLE
Varrock npc spawns & Spawns.Kt Change

### PR DESCRIPTION
**Pull request by WizardJesse1**
Converted varrock npcs .rb to .kts, added a second method so we can control the height of the npc spawns

### CODE REVIEW

File: game/src/plugins/entity/spawn/src/spawn.kt [hunk 1]
```diff
@@ -13,3 +13,7 @@ object Spawns {
 fun npc_spawn(name: String, x: Int, y: Int, id: Int? = null) {
```
Comment (ryleykimmel) [inline game/src/plugins/entity/spawn/src/spawn.kt]: Remove this method - add default height of in your newly added ```npc_spawn``` method.

Also consider adding Direction as a parameter and setting its default to NORTH

Comment (ryleykimmel) [inline game/src/plugins/entity/spawn/src/spawn.kt]: See https://kotlinlang.org/docs/reference/functions.html#default-arguments for information on default args

File: game/src/plugins/locations/tutorial-island/src/tutorial-island-npcs.kts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+// pretends to be underground.
+
+npc_spawn(":giant_rat", id = 87, x = 3105, y = 9514)
+npc_spawn(":giant_rat", id = 87, x = 3105, y = 9517)
+npc_spawn(":giant_rat", id = 87, x = 3106, y = 9514)
+npc_spawn(":giant_rat", id = 87, x = 3104, y = 9514)
+npc_spawn(":giant_rat", id = 87, x = 3105, y = 9519)
+npc_spawn(":giant_rat", id = 87, x = 3109, y = 9516)
+npc_spawn(":giant_rat", id = 87, x = 3108, y = 9520)
+npc_spawn(":giant_rat", id = 87, x = 3102, y = 9517)
```
Comment (cubeee) [inline game/src/plugins/locations/tutorial-island/src/tutorial-island-npcs.kts]: Colon left in names

**Comment by ryleykimmel** [comment]
LGTM cc @Major- @garyttierney

**Comment by Major-** [comment]
Nice one, thanks @WizardJesse1.

Delegating to @garyttierney as he's doing the npc spawn stuff. Also we need to decide on a consistent format for plugin names (capitalised? all lowercase is a bit grim) but that doesnt necessarily need to block this PR

**Comment by garyttierney** [comment]
Merged in 766f94c72ab1ebce1afbf7feaa13ef6e2fe88a6a, thanks.

**Pull request closed without merging**